frappe/utils/safe_exec.py

```diff
@@ -110,7 +110,8 @@ def get_safe_globals():
 			socketio_port=frappe.conf.socketio_port,
 			get_hooks=frappe.get_hooks,
 			sanitize_html=frappe.utils.sanitize_html,
-			log_error=frappe.log_error
+			log_error=frappe.log_error,
+			cache=frappe.cache 
```

Comment:
Let us add specific methods rather than exposing the full module, there might be security flaws here.